engine/wikiRunner.ts

```diff
@@ -11,13 +12,17 @@ export abstract class WikiRunner extends Runner {
     [".js", "javascript"], [".html", "html"],
     [".scss", "css"], [".asciidoc", "asciidoc"], ]);
     protected readonly INSTALLED_TOOLS: string = "installedTools";
+    public CommandCntMap = new Map();
```

Comment:
Start variable names with lower case letters

Reply:
Changed it to lower case